jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/InvalidContainerVolumeException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.plugins.common;
+
+/** Thrown when list of volume's directories contains invalid paths */
+public class InvalidContainerVolumeException extends Exception {
```

Comment:
Hi, do you mind making this in the same format as the other exceptions regarding invalid paths? Like https://github.com/GoogleContainerTools/jib/pull/1266/files#diff-a90e603b657d4bf5aca92fe07322a32e - where there is a `getInvalidPathValue` method and the exception is caught at the plugin level so that the parameter name can be in the exception message (`container.volume` or `<container><volume>`).

Reply:
No problem at all, I think it looks better now following the same format.